packages/react-devtools-extensions/src/parseHookNames/loadSourceAndMetadata.js

```diff
@@ -335,6 +348,24 @@ function fetchFile(url: string): Promise<string> {
   });
 }
 
+export function hasNamedHooks(hooksTree: HooksTree): boolean {
```

Comment:
where do we use this function?

Reply:
We will eventually use this in the UI to avoid showing the 🪄 for components that have no hooks. Can't use it yet though (since we can't load code from react-devtools-extensions from react-devtools-shared).